src/docs/source/includes/_deployment.md

```diff
@@ -246,6 +246,103 @@ values as described [above](#provisioning)
     kubectl get servicebindings -A
     ```
 
+## API Gateway, OCI Functions and Email Delivery
+
+Note that this is optional. If you don't want to configure Email Delivery and deploy an API Gateway and the function, skip to the [Deployment](#deployment) section. 
+
+### Configure Email Delivery
+
+If you are planning to use the API gateway and Oracle Functions, to send emails using OCI Email Delivery you need to configure an approved sender first.
+
+1. From the OCI console, click **Email Delivery** -> **Email Approved Sender**
+1. Click the **Create Approved Sender**
+1. Enter the email address, for example: `mushop@example.com`
+1. Click **Create Approved Sender**
+
+>Note: if you have your own domain, you can enter a different address (e.g.`mushop@[yourdomain.com]`) and also configure SPF record for the sender. This involves adding a DNS record to your domain. You can follow [these](https://docs.cloud.oracle.com/iaas/Content/Email/Tasks/configurespf.htm) instructions to set up SPF.
+
+Next, you need to generate the SMTP credentials that will allow you to log in to the SMTP server and send the email. Follow the [Generate SMTP Credentails for a User](https://docs.cloud.oracle.com/iaas/Content/Email/Tasks/generatesmtpcredentials.htm) to get the SMTP host, port, username and password.
+
+The SMTP credentails (host, port, username and password) and the approved sender email address (e.g. `mushop@example.com`) will be provided to the function as configuration values later, so make sure you save these values somewhere.
+
+### Configure function applicaiton
+
+Each function needs to live inside of an application. You can create a new application either through the console, API or the Fn CLI. An application has a name (e.g. `mushop-app`) and the VCN and a subnet in which to run the functions. The one guideline here is to pick the subnets that are in the same region as the Docker registry you specified in your context YAML earlier - check these [docs](https://docs.cloud.oracle.com/iaas/Content/Functions/Tasks/functionscreatingapps.htm) for more information.
```

Comment:
Are there any included links to install the fn CLI?

Reply:
added the section